extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -278,6 +295,16 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * Receives any response that may be sent from the cross-domain iframe
+   * @param {String} msg The response message from the iframe
+   * was specified in the amp-analytics config
+   */
+  processCrossDomainIframeResponse_(msg) {
+    // Used in ../../../src/anchor-click-interceptor.js
+    this.win.document.mostRecentCrossDomainIframeResponse = msg;
```

Comment:
Done. Note that it's enclosed in the anchor click listener not amp-analytics, to avoid anchor click listener having a dependency on amp-analytics which lives in extensions.